src/utils/aeternity.js

```diff
@@ -73,6 +73,12 @@ export const initSdk = async () => {
   store.commit('setBackendStatus', true);
 };
 
+export const getClient = async () => {
```

Comment:
Can you get sdk from `store.state.sdk`? You can use `$watchUntilTruly` to ensure that it is available.

Reply:
removed for now